### PR TITLE
Issue 71

### PR DESCRIPTION
**Pull request by saltpy**
I think this is a fix for #71 - let me know if I need to do anything else.

### CODE REVIEW

File: appium/webdriver/webdriver.py [hunk 1]
```diff
@@ -693,6 +693,19 @@ def toggle_location_services(self):
...
 
+    def set_location(self, latitude, longitude, altitude):
+        """Set the location of the device
+
+        :Args:
+         - latitude - String or numeric value between -90.0 and 90.00
+         - longitude - String or numeric value between -180.0 and 180.0
+         - altitude - String or numeric value
+        """
+        data = {'location': {"latitude": str(latitude),
```
Comment (imurchie) [inline appium/webdriver/webdriver.py]: Can you format this literal in a more readable way, and standardize the quotes?

```python
data = {
    'location': {
        'latitude': str(latitude),
        'longitude': str(longitude),
        'altitude': str(altitude)
    }
}
```

**Comment by imurchie** [comment]
Once comment. Then can you squash into a single commit?

**Comment by saltpy** [comment]
@imurchie I think that is done now - first one of these on github so thank you for being patient with the errors etc

**Comment by imurchie** [comment]
Thanks!

**Pull request merged**